paddle/fluid/pybind/gloo_context_py.cc

```diff
@@ -0,0 +1,111 @@
+/* Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
same above

Reply:
Yes, I will fix the issue in the pr: https://github.com/PaddlePaddle/Paddle/pull/26586